src/index.js

```diff
@@ -63,7 +47,7 @@ const graphServer = apolloGraphServer(createConfig());
 
 const useSentry = async (req, res, next) => {
   Sentry.addBreadcrumb({
-    category: 'that-api-sessions',
+    category: 'root',
```

Comment:
@csell5  Will this still record under/tagged as that-api-sessions for `thatapp` tag in Sentry?

Reply:
It shows up in the stack trace for the call not the event

Reply:
It shows up in the stack trace for the call not the event